edgelet/edgelet-docker/src/lib.rs

```diff
@@ -13,9 +13,29 @@
 
 // mod client;
 mod error;
+mod image_prune_data;
 mod module;
 mod runtime;
 
 pub use error::Error;
+pub use image_prune_data::ImagePruneData;
 pub use module::{DockerModule, MODULE_TYPE};
 pub use runtime::{init_client, DockerModuleRuntime};
+
+use tokio::sync::mpsc::UnboundedSender;
+
+use edgelet_core::{ModuleAction, ModuleRuntime};
+use edgelet_settings::RuntimeSettings;
+
+#[async_trait::async_trait]
+pub trait MakeModuleRuntime {
```

Comment:
Would creating a new image-gc crate as suggested in my previous comment remove the circular dependency that caused you to move `MakeModuleRuntime` from `edgelet-core` to `edgelet-docker`?

Reply:
I don't remember...